### PR TITLE
JBrowse missing pyyaml

### PR DESCRIPTION
**Pull request by erasche**
Reported by @pvanheus

### CODE REVIEW

File: packages/package_python_2_7_pyyaml_3_11/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            <actions>
+                <action type="setup_python_environment">
+                    <repository name="package_python_2_7" owner="iuc">
+                        <package name="python" version="2.7" />
+                    </repository>
+                    <package>https://pypi.python.org/packages/source/P/PyYAML/PyYAML-3.11.tar.gz#md5=f50e08ef0fe55178479d3a618efe21db</package>
+                </action>
+                <action type="set_environment">
+                    <environment_variable action="prepend_to" name="PYTHONPATH">$INSTALL_DIR</environment_variable>
+                    <environment_variable action="prepend_to" name="PATH">$INSTALL_DIR/bin</environment_variable>
```
Comment (nsoranzo) [inline packages/package_python_2_7_pyyaml_3_11/tool_dependencies.xml]: I don't think PyYAML is installing anything in `bin/` .

Comment (erasche) [inline packages/package_python_2_7_pyyaml_3_11/tool_dependencies.xml]: Probably not. This was all auto-generated ;) https://github.com/bgruening/notebooks/pull/7

I'll remove this line.

Comment (bgruening) [inline packages/package_python_2_7_pyyaml_3_11/tool_dependencies.xml]: It's usually my fault ;)

Comment (erasche) [inline packages/package_python_2_7_pyyaml_3_11/tool_dependencies.xml]: [Git blame](https://github.com/erasche/notebooks/blame/perl-upgrade/Python/python_module_generator.py) says it's me, you're off the hook @bgruening.

**Comment by nsoranzo** [comment]
:+1:

**Pull request merged**